nucypher/utilities/events.py

```diff
@@ -52,6 +52,8 @@ def write_events_to_csv_file(csv_file: str,
             event_row = OrderedDict()
             event_row['event_name'] = event_name
             event_row['block_number'] = event_record.block_number
+            event_row['timestamp'] = event_record.timestamp
```

Comment:
Let's be explicit about the timestamp format since I think apps (eg. excel) has its own epoch.

```suggestion
            event_row['unix_timestamp'] = event_record.timestamp
```